From lieb@canonical.com  Mon Aug 17 12:53:35 2009
From: Jim Lieb <lieb@canonical.com>
Date: Wed, 12 Aug 2009 14:54:13 -0700
Subject: Staging: vt665x: remove tpci.h file in vt6656
To: Greg KH <greg@kroah.com>, Forest Bond <forest@alittletooquiet.net>, Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Cc: jim.lieb@canonical.com, Jim Lieb <lieb@canonical.com>
Message-ID: <1250114056-13843-12-git-send-email-lieb@canonical.com>


Remove tpci.h.  It is not even used in the usb version
Minor cleanup of janitor work missed earlier.

Signed-off-by: Jim Lieb <lieb@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/vt6656/device.h |   31 ----------
 drivers/staging/vt6656/tpci.h   |  113 ----------------------------------------
 2 files changed, 144 deletions(-)
 delete mode 100644 drivers/staging/vt6656/tpci.h

--- a/drivers/staging/vt6656/device.h
+++ b/drivers/staging/vt6656/device.h
@@ -94,50 +94,19 @@
 // device specific
 //
 
-#if !defined(_KCOMPAT_H)
 #include "kcompat.h"
-#endif
-
-#if !defined(__DEVICE_CONFIG_H)
 #include "device_cfg.h"
-#endif
-
-#if !defined(__TTYPE_H__)
 #include "ttype.h"
-#endif
-#if !defined(__80211HDR_H__)
 #include "80211hdr.h"
-#endif
-#if !defined(__TETHER_H__)
 #include "tether.h"
-#endif
-#if !defined(__WMGR_H__)
 #include "wmgr.h"
-#endif
-#if !defined(__WCMD_H__)
 #include "wcmd.h"
-#endif
-#if !defined(__MIB_H__)
 #include "mib.h"
-#endif
-#if !defined(__SROM_H__)
 #include "srom.h"
-#endif
-#if !defined(__RC4_H__)
 #include "rc4.h"
-#endif
-#if !defined(__TPCI_H__)
-#include "tpci.h"
-#endif
-#if !defined(__DESC_H__)
 #include "desc.h"
-#endif
-#if !defined(__KEY_H__)
 #include "key.h"
-#endif
-#if !defined(__CARD_H__)
 #include "card.h"
-#endif
 
 /*---------------------  Export Definitions -------------------------*/
 #define VNT_USB_VENDOR_ID                     0x160A
--- a/drivers/staging/vt6656/tpci.h
+++ /dev/null
@@ -1,113 +0,0 @@
-/*
- * Copyright (c) 1996, 2003 VIA Networking Technologies, Inc.
- * All rights reserved.
- *
-
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, write to the Free Software Foundation, Inc.,
- * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
- *
- *
- * File: tpci.h
- *
- * Purpose: PCI routines
- *
- * Author: Tevin Chen
- *
- * Date: May 21, 1996
- *
- */
-
-#ifndef __TPCI_H__
-#define __TPCI_H__
-
-#include "ttype.h"
-
-/*---------------------  Export Definitions -------------------------*/
-#define MAX_PCI_BUS             4       // max. # of PCI bus that we support
-#define MAX_PCI_DEVICE          32      // max. # of PCI devices
-
-
-//
-// Registers in the PCI configuration space
-//
-#define PCI_REG_VENDOR_ID       0x00    //
-#define PCI_REG_DEVICE_ID       0x02    //
-#define PCI_REG_COMMAND         0x04    //
-#define PCI_REG_STATUS          0x06    //
-#define PCI_REG_REV_ID          0x08    //
-#define PCI_REG_CLASS_CODE      0x09    //
-#define PCI_REG_CACHELINE_SIZE  0x0C    //
-#define PCI_REG_LAT_TIMER       0x0D    //
-#define PCI_REG_HDR_TYPE        0x0E    //
-#define PCI_REG_BIST            0x0F    //
-
-#define PCI_REG_BAR0            0x10    //
-#define PCI_REG_BAR1            0x14    //
-#define PCI_REG_BAR2            0x18    //
-#define PCI_REG_CARDBUS_CIS_PTR 0x28    //
-
-#define PCI_REG_SUB_VEN_ID      0x2C    //
-#define PCI_REG_SUB_SYS_ID      0x2E    //
-#define PCI_REG_EXP_ROM_BAR     0x30    //
-#define PCI_REG_CAP             0x34    //
-
-#define PCI_REG_INT_LINE        0x3C    //
-#define PCI_REG_INT_PIN         0x3D    //
-#define PCI_REG_MIN_GNT         0x3E    //
-#define PCI_REG_MAX_LAT         0x3F    //
-
-#define PCI_REG_MAX_SIZE        0x100   // maximun total PCI registers
-
-
-//
-// Bits in the COMMAND register
-//
-#define COMMAND_BUSM        0x04        //
-#define COMMAND_WAIT        0x80        //
-
-/*---------------------  Export Types  ------------------------------*/
-
-/*---------------------  Export Macros ------------------------------*/
-
-// macro MAKE Bus Dev Fun ID into WORD
-#define MAKE_BDF_TO_WORD(byBusId, byDevId, byFunId) \
-    MAKEWORD(                                       \
-        ((((BYTE)(byDevId)) & 0x1F) << 3) +         \
-            (((BYTE)(byFunId)) & 0x07),             \
-        (byBusId)                                   \
-        )
-
-#define GET_BUSID(wBusDevFunId) \
-    HIBYTE(wBusDevFunId)
-
-#define GET_DEVID(wBusDevFunId) \
-    (LOBYTE(wBusDevFunId) >> 3)
-
-#define GET_FUNID(wBusDevFunId) \
-    (LOBYTE(wBusDevFunId) & 0x07)
-
-
-/*---------------------  Export Classes  ----------------------------*/
-
-/*---------------------  Export Variables  --------------------------*/
-
-/*---------------------  Export Functions  --------------------------*/
-
-
-
-
-#endif // __TPCI_H__
-
-
